mypy/stubtest.py

```diff
@@ -1256,6 +1256,21 @@ def verify_paramspecexpr(
         return
 
 
+def _is_django_cached_property(runtime: Any) -> bool:  # pragma: no cover
+    # This is a special case for
+    # https://docs.djangoproject.com/en/5.2/ref/utils/#django.utils.functional.cached_property
+    # This is needed in `django-stubs` project:
+    # https://github.com/typeddjango/django-stubs
+    try:
+        from django.utils.functional import cached_property  # type: ignore[import-not-found]
+    except Exception:
```

Comment:
Should this be `except Exception` and not `except ImportError`?

Reply:
This is `except Exception` to be extra safe.

Reply:
I wonder if this would be cleaner if we'd check the fully qualified name of the type instead of using `isinstance`? For example, we might check `repr(type(runtime))`. This way we wouldn't need to import anything.

Reply:
I double checked @JukkaL's suggestion. It works! Pushed my change.